spec/rubocop/cli/auto_gen_config_spec.rb

```diff
@@ -951,6 +951,32 @@ def a; end
       YAML
     end
 
+    context 'when configurable parameter is a deprecated parameter' do
```

Comment:
Is this a typo?

```suggestion
    context 'when configurable parameter is a duplicated parameter' do
```

Reply:
This problem occurred with a deprecated option, so deprecated is correct, but it is confusing, so I changed it to obsoletion.